README.md

```diff
@@ -50,71 +67,82 @@ Let us infer the values of the unknown parameters in our model by running MCMC u
 >>> mcmc = MCMC(nuts_kernel, num_warmup=500, num_samples=1000)
 >>> rng_key = random.PRNGKey(0)
 >>> mcmc.run(rng_key, J, sigma, y=y, extra_fields=('potential_energy',))
+
 ```
 
 We can print the summary of the MCMC run, and examine if we observed any divergences during inference. Additionally, since we collected the potential energy for each of the samples, we can easily compute the expected log joint density.
 
 ```python
 >>> mcmc.print_summary()
-
+<BLANKLINE>
```

Comment:
Why `<BLANKLINE>` - is this meant to be rendered differently?

Reply:
Oh..so `<BLANKLINE>` is for doctest?

Reply:
The output of [doctest](https://docs.python.org/3/library/doctest.html#how-are-docstring-examples-recognized) expects <BLANKLINE> here:  "If expected output does contain a blank line, put <BLANKLINE> in your doctest example each place a blank line is expected." Do you know a trick to remove it?

Reply:
One option is to add `# doctest: +SKIP` to the specific lines. I think these are likely to be flaky so its a good idea to skip asserting on the actual output in these cases.

Reply:
Yes, I think we can do so and add this doctest to travis.